README.md

```diff
@@ -1,3 +1,10 @@
 #Emender Tests
 
 A generic set of tests using Emender.
+
+## Asciidoctor Tests and Docbook Tests
+
+These folders containers a set of generic tests that can be re-used and/or modified for use with asciidoctor or docbook projects who want to integrate emender.
+
+## Test
+This folder contains a Dockerfile and supporting scripts to run all the provided tests against a set of example content.
```

Comment:
Would you mind linking to [Docker documentation](https://docs.docker.com/engine/installation/linux/) for information on how to install and configure it on the host? This might also be a good place to explain how to run the scripts, for example:

  1. Change to the `test` directory.

        $ cd test/

  2. Run the `build.sh` script to build a new docker image with *emender-tests*:

        $ ./build.sh

  3. Edit the `runAllTests.sh` file and replace the value of the `REPO_PATH` variable with the absolute path to the `emender-tests` directory, for example:

        REPO_PATH="/home/arya/emender-tests"

  4. Run the `runAllTests.sh` scripts to verify that provided Emender tests work as expected:

        $ ./runAllTests.sh